synapse/lib/cell.py

```diff
@@ -2394,9 +2396,9 @@ async def _getSpawnLogConf(self):
             return conf
         return s_common._getLogConfFromEnv()
 
-    def modCellConf(self, conf):
+    def updateCellModConf(self, conf):
```

Comment:
Because this function updates both the running and the persistent config, it should probably keep the original name `modCellConf` The fact that the API changes the config file in one spot or another is mostly irrelevant :)